server/raft_transport.go

```diff
@@ -190,8 +190,10 @@ func (t *rpcTransport) processQueue(raftNodeID proto.RaftNodeID) {
 		}
 
 		if !client.IsHealthy() {
-			log.Warningf("raft client for node %d unhealthy", nodeID)
-			return
+			if log.V(1) {
```

Comment:
In general we do want to drop raft messages if a peer is known to be unhealthy. There's no point in unleashing a ton of queued heartbeats as soon as a node comes back up.